content/20.treatments.md

```diff
@@ -505,6 +504,42 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
+These findings are supported by a recently published meta-analysis [@doi:10.4187/respcare.07097].
+There is a significant risk of ventilator-associated pneumonia in COVID-19 patients [@doi:10.6084/m9.figshare.12340496], but it can be challenging for clinicians to diagnose this infection due to the fact that severe COVID-19 infection present with the symptoms of pneumonia [@doi:10.1186/s13054-020-03013-2].
+Therefore, an effective prophylactic therapy for ventilator-associated pneumonia in severe COVID-19 patients would be of significant therapeutic value.
+
+Probiotics are generally synonymous with the treatment of gastrointestinal issues due to their supposed anti-inflammatory and immunomodulatory effects [@doi:10.1016/S2352-4642(19)30182-8]. 
+Notably, gastrointestinal symptoms commonly occur in COVID-19 patients [@doi:10.1053/j.gastro.2020.03.020], and the ACE2 receptor is highly expressed in enterocytes of the ileum and colon, suggesting that these organs may be a potential route of infection [@doi:10.1101/2020.01.30.927806; @doi:10.1038/cr.2016.152]. 
+Indeed, SARS-CoV-2 viral RNA has been detected in human feces [@doi:10/ggq8zp] and fecal-oral transmission of the virus has not yet been ruled out [@doi:10/ggpx7s]. 
+Rectal swabs of some SARS-CoV-2 positive pediatric patients persistently tested positive for several days despite negative nasopharyngeal tests, indicating the potential for fecal viral shedding [@doi:10.1038/s41591-020-0817-4].
+However, there is conflicting evidence for the therapeutic value of various probiotics against the incidence or severity of gastrointestinal symptoms in viral or bacterial infections such as gastroenteritis [@doi:10.1007/s00203-017-1400-3; @doi:10.1056/NEJMoa1802597]. 
+Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19, a hypothesis that several clinical trials are now preparing to investigate [@clinicaltrials:NCT04420676; @clinicaltrials:NCT04390477; @clinicaltrials:NCT04366180]. 
+
+**Summary.**
+Generally, the efficacy of probiotic use is a controversial topic among scientists.
+In Europe, EFSA has banned the term probiotics on products labels, which has elicited either criticism for EFSA or support for probiotics from researchers in the field [@doi:10.1016/j.bpg.2015.12.001; @doi:10.1017/S000711451100287X; @doi:10.1038/nrgastro.2014.66]. 
+This is due to the hyperbolical claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy. 
```

Comment:
```suggestion
This is due to the hyperbolic claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy. 
```
[Apparently hyperbolical isn't wrong but is just not often used?](https://english.stackexchange.com/questions/238906/hyperbolic-vs-hyperbolical) TIL!

Reply:
now that I read it back it seems aloof haha